public/app/features/browse-dashboards/components/NameCell.tsx

```diff
@@ -27,7 +27,7 @@ export function NameCell({ row: { original: data }, onFolderClick, treeID }: Nam
   const { item, level, isOpen } = data;
   const childrenByParentUID = useChildrenByParentUIDState();
   const isLoading = isOpen && !childrenByParentUID[item.uid];
-  const iconName = getIconForKind(data.item.kind, isOpen);
+  const iconName = getIconForKind(data.item.kind, isOpen, data.item);
```

Comment:
Yeah, that was my first attempt, but it's hard because this is used in Search where we don't have a DashboardViewItem. I've settled for a middleground where it's `itemOrKind: string | DashboardViewItem`